src/main/frontend/components/header.cljs

```diff
@@ -58,7 +58,7 @@
   [{:keys [on-click]}]
   (ui/with-shortcut :ui/toggle-left-sidebar "bottom"
     [:button.#left-menu.cp__header-left-menu.button.icon
-     {:title "Toggle left menu"
+     {:title (t :header/toggle-left-sidebar)
```

Comment:
@logseq-cldwalker please check. I changed according to this conversation https://github.com/logseq/logseq/pull/9588#discussion_r1218522686.